lib/postprocessing/image.js

```diff
@@ -450,7 +462,13 @@ async function write(img, outfile) {
  * @param {Object} directories working directories of the current activation
  */
 async function imagePostProcess(intermediateRendition, rendition, directories) {
+
+    if (intermediateRendition._intermediateMetadata.format === "SVG") {
+        intermediateRendition.path = await convertSvg(intermediateRendition.path, intermediateRendition.instructions);
+    }
+
     const img = await read(intermediateRendition.path);
+    //img.setFormat(rendition.instructions.fmt);
```

Comment:
@DrGirlfriend I'd recommend uncommenting that too, but also at the end, when we have svg working.
(This helps with CMYK colorspace handling, by setting defaults if the format that was set can't handle CMYK).

Reply:
(done)